source/extensions/common/tap/tap_config_base.cc

```diff
@@ -277,6 +281,90 @@ void FilePerTapSink::FilePerTapSinkHandle::submitTrace(
   }
 }
 
+// The implemented code for UDP sink
+UdpTapSink::UdpTapSink(const envoy::extensions::tap_sinks::udp_sink::v3::UdpSink& config)
+    : config_(config) {
+  if (config_.udp_address().protocol() != envoy::config::core::v3::SocketAddress::UDP) {
+    ENVOY_LOG_MISC(warn, "{}: Only suport UDP and invalid protocol", __func__);
+    return;
+  }
+
+  // Verify the address (ipv4/ipv6)
```

Comment:
style-nit: comments should end with a '.'

Reply:
Ok, Added